android/cli/commands/_cleanModule.js

```diff
@@ -12,14 +12,28 @@
 */
 
 'use strict';
-const path = require('path');
-const fs = require('fs-extra');
+
 const appc = require('node-appc');
+const fs = require('fs-extra');
+const GradleWrapper = require('../lib/gradle-wrapper');
+const path = require('path');
 const __ = appc.i18n(__dirname).__;
 
-exports.run = function run(logger, config, cli, finished) {
+exports.run = async function run(logger, config, cli, finished) {
 	const projectDir = cli.argv['project-dir'];
 
+	// Run the gradle "clean" task if possible.
+	// This makes the gradle daemon release its file locks so that they can be deleted on Windows.
+	try {
+		const gradlew = new GradleWrapper(path.join(projectDir, 'build'));
+		gradlew.logger = logger;
+		if (await gradlew.hasWrapperFiles()) {
```

Comment:
Upon testing, `hasWrapperFiles()` returns `false` and `appc ti clean` ends with
```
(node:21360) UnhandledPromiseRejectionWarning: Error: EBUSY: resource busy or locked, unlink 'C:\Users\gmathews\github\appcelerator.https\android\build\local.properties'
```

Reply:
I'm not sure how my project got into that state, after manually killing the process that had the lock and re-testing, it's working fine now.